sstables/exceptions.hh

```diff
@@ -45,14 +45,29 @@ struct bufsize_mismatch_exception : malformed_sstable_exception {
     {}
 };
 
-class compaction_stop_exception : public std::exception {
+class compaction_job_exception : public std::exception {
     sstring _msg;
 public:
-    compaction_stop_exception(sstring ks, sstring cf, sstring reason) :
-        _msg(format("Compaction for {}/{} was stopped due to: {}", ks, cf, reason)) {}
+    compaction_job_exception(sstring msg) : _msg(std::move(msg)) {}
     const char *what() const noexcept {
         return _msg.c_str();
     }
 };
 
+// Indicates that compaction was stopped via an external event,
+// E.g. shutdown or api call.
+class compaction_stopped_exception : public compaction_job_exception {
```

Comment:
if we're switching to past tense, perhaps abort should be aborted instead, for symmetry.

Reply:
ok